app/server.R

```diff
@@ -97,11 +102,15 @@ shinyServer(function(input, output, session) {
     
     output$number_county <- renderPlotly({
         if(input$selectedNumber == "30"){
-            mainplot_data <- number_county_30 
+            current_plot <- number_county_30 
         } 
         else {
-            mainplot_data <- number_county_50
+            current_plot <- number_county_50
         }
+        current_plot %>% 
+            ggplotly() %>%
+            plotly_disable_zoom() %>%
+            plotly_hide_modebar
```

Comment:
I think it may be better to bundle these two functions (or the routine) since it's used repeatedly. I'm keeping it for now because we are not repeating 3 or more times.